### PR TITLE
UX: be specific if disko-install failed or succeeded

### PR DESCRIPTION
**Pull request by DavHau**
I was running into a problem where disko install failed but the logs didn't indicate that.

### CODE REVIEW

File: disko-install [hunk 1]
```diff
@@ -244,4 +244,10 @@ main() {
   nixos-install --no-channel-copy --no-root-password --system "$nixos_system" --root "$mountPoint"
 }
 
-main "$@"
+if main "$@"; then
+  echo "disko-install succeeded" >&2
```
Comment (iFreilicht) [inline disko-install:248]: Shouldn't this go to stdout?

Comment (Mic92) [reply disko-install:248]:
```suggestion
  echo "disko-install succeeded"
```

**Comment by iFreilicht** [comment]
@mergify squash

**Comment by iFreilicht** [comment]
@mergify queue

**Pull request merged**